seed/models/analyses.py

```diff
@@ -30,6 +31,7 @@ class Analysis(models.Model):
     COMPLETED = 70
 
     STATUS_TYPES = (
+        (PENDING_CREATION, 'Pending Creation'),
```

Comment:
Needed some way to indicate "the preparation of the analysis has not been started" to prevent more than one celery task to start creating the analysis.